src/Command/ComputeAnswersCommand.php

```diff
@@ -58,6 +58,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
 
         /** @var Group $group */
         foreach ($groups as $group) {
+            try {
             $questionnaire = $this->quapRepository->getQuestionnaireByGroup($group);
             $questions = $this->questionRepository->findEvaluableByQuestionnaire($questionnaire);
```

Comment:
@SebastianStorz please format this code correctly and add tabs here (otherwise the pipeline won't succeed due to our linting rules)